wls-gui-wahllokalsystem/src/components/broadcast/TheBroadcastReadConfirmationDialog.vue

```diff
@@ -0,0 +1,70 @@
+<template>
+  <v-dialog
+    :model-value="hasBroadcastnachricht"
+    max-width="400px"
+    persistent
+    data-test="dialog-show-broadcast-nachricht"
+  >
+    <v-card>
+      <v-card-title>
```

Comment:
```suggestion
      <v-card-title class="d-flex align-center">
```
damit das icon und der text vertikal zentriert sind (es fällt mehr auf wenn man dem icon eine andere größe gibt, dass sie nciht zentriert sind)

Reply:
bin immer noch dafür das zu zentrieren (: